### ISSUE TITLE
NTR: Abdominal wall biopsy: desmoid tumors (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9138]
New term request Abdominal wall biopsy: desmoid tumors (elthat_100307031900): Fibrous neoplasm of musculoaponeurotic structures  UMLS: 0206646

**Comment by pnrobinson** [comment]
@MickeySegal
what is the medical context?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/135290 and https://www.ncbi.nlm.nih.gov/books/NBK1345/ and https://www.omim.org/entry/615848

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001034
- label: Desmoid tumor
- parents: [Soft tissue neoplasm (HP:0031459)](https://hpo.jax.org/app/browse/term/HP:0031459)
- pmid: PMID:37436594|PMID:32004793
- def: Desmoid tumors are rare, locally aggressive, fibroblastic soft-tissue tumors that are characterized by infiltrative growth and tendency to recur. Desmoid tumors do not metastasize but can arise in any part of the body: extra-abdominal (appear in the head and neck, chest, extremities), and abdominal wall or intra-abdominal (originating in tissue that connects abdominal organs).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9138
- synonyms: Aggressive fibromatosis(NONE)|Desmoid-type fibromatosis(NONE)

**Comment by MickeySegal** [comment]
I see that HPO already has https://hpo.jax.org/browse/term/HP:0100245 Desmoid tumors.  So it seems like the new term is not needed.

**Comment by pnrobinson** [comment]
check this

**Comment by pnrobinson** [comment]
@MickeySegal 
https://hpo.jax.org/browse/term/HP:0100245  intended to be
Gastrointestinal desmoid tumor
I have relabled!
